core-libs/setup/ssr/engine-decorator/ng-express-engine-decorator.ts

```diff
@@ -42,10 +45,7 @@ export class NgExpressEngineDecorator {
 
 export function decorateExpressEngine(
   ngExpressEngine: NgExpressEngine,
-  optimizationOptions: SsrOptimizationOptions | null = {
-    concurrency: 20,
-    timeout: 3000,
-  }
+  optimizationOptions: SsrOptimizationOptions | null = defaultSsrOptimizationOptions
```

Comment:
Do we _really_ need to provide a default value here? We're already `{...spreading}` the default options [in OptimizedSsrEngine in line 66](https://github.com/SAP/spartacus/pull/15924/files#diff-1db882c5fb0276ec1884fe195efa449fcc93d591c38b783ca45396de81dbf36fR66)